packages/kotti-ui/source/kotti-filter/types.ts

```diff
@@ -0,0 +1,155 @@
+import { Kotti } from '../types'
+import { SpecifyRequiredProps } from '../types/utilities'
+
+export namespace KottiFilter {
+	export enum ButtonLinkType {
+		DANGER = 'danger',
+		PRIMARY = 'primary',
+	}
+
+	export enum FilterType {
+		BOOLEAN = 'BOOLEAN',
+		DATE_RANGE = 'DATE_RANGE',
+		MULTI_ENUM = 'MULTI_ENUM',
+		NUMBER = 'NUMBER',
+		SEARCH = 'SEARCH',
+		SINGLE_ENUM = 'SINGLE_ENUM',
+		STRING = 'STRING',
+	}
+
+	export namespace Operation {
+		export enum Boolean {
+			EQUAL = 'EQUAL',
+		}
+
+		export enum Common {
```

Comment:
While I appreciate the effort for re-usability, I don’t think this is a good place for it, as it’s not discoverable and leads to easy-to-miss issues where only `Operation.String` is handled but `Operation.Common` isn’t.

This should either somehow inherit from the `Common` one, or it should be duplicated across all different types.

I have no good suggestion on how to handle the translations, besides copy-paste. Don’t think it’s a major issue though.